File: CS/WpfApplication1/Customer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using DevExpress.Xpo;
using System.Windows;
using System.ComponentModel;

namespace WpfApplication1 {
    public class ContactItem {
        protected string _Name;
        public string Name {
            get { return this._Name; }
            set { this._Name = value; }
        }
        public decimal Salary { get; set; }
        public DateTime Date { get; set; }
    }

}

File: VB/WpfApplication1/Customer.vb
Imports System.ComponentModel

Namespace WpfApplication1

    Public Class ContactItem

        Protected _Name As String

        Public Property Name As String
            Get
                Return _Name
            End Get

            Set(ByVal value As String)
                _Name = value
            End Set
        End Property

        Public Property Salary As Decimal

        Public Property [Date] As Date
    End Class
End Namespace
